### DISCUSSION TITLE
I want to embed Jbrowse2 into my H5 page. I have some issues loading the file locally and would like to consult with you.

### DISCUSSION THREAD

**Discussion by Opticalproperty** [GMOD/jbrowse-components#3650] [Q&A] [answered]
Hello, jbrowser developers！I encountered some issues while using jbrowse2,I hope to receive your reply.
I tried centOS8 and ubuntu systems,Through the document's ‘JBrowse web setup using the CLI’.
I want to display my two files, one with a suffix of. vcf and the other with a suffix of. bed, on jbrowser. However, after manually modifying config.json in volox, I can find the current file on the right side of the page with the file name. However, after opening it, the page appears empty and does not display the correct file information. Next, I tried to import the file directly on the page, but my local project is still empty.
![image](https://user-images.githubusercontent.com/116154971/232716038-68eb6820-0246-40db-aa88-da17aa746e24.png)
If I only want to display my files on jbrowser, how do I need to configure them specifically?
Thank you.

**Comment by cmdcolin** [comment]
Hello,
I would recommend creating a new config rather than editing the volvox config (which is just sort of our example test data). I will describe below how to do that below...

Your track name suggests you are using human, so probably you will start with a FASTA file for human (e.g. hg19.fa or hg38.fa) and you can use

```
samtools faidx /path/to/hg19.fa ## first create fasta index (faidx)
jbrowse add-assembly /path/to/hg19.fa --load copy --out /path/to/jbrowse2 # just the root jbrowse 2 directory, not the e.g. /path/to/jbrowse2/test_data/volvox/
```

Then you can add your tracks...You can add plaintext BED or VCF if they are small-ish file sizes, but for larger sizes, I recommend using tabix to compress and index them

So then you use

```
jbrowse add-track /path/to/yourfile.vcf --out /path/to/jbrowse2 --load copy # if it is a plaintext vcf
jbrowse add-track /path/to/yourfile.vcf.gz --out /path/to/jbrowse2 # if it is a tabix gzipped vcf
```

**Reply by Opticalproperty** [reply]
Thank you！

I'm facing an issue with importing a .bed file through jbrowsecli, as it reminds me to specify a type but I cannot find the type specified in the documentation. How can I resolve this?

Also, I'm trying to configure direct access to local addresses like [http://localhost:3000?data=my_data/config.json](http://localhost:3000/?data=my_data/config.json). When I access this address, how can I make sure that my genome file is displayed by default?

**Reply by cmdcolin** [reply]
Looks like the cli doesnt have plaintext bed yet so can make a tabix bed and it'll detect the .bed.gz file extension 

```
bgzip file.bed
tabix file.bed.gz
jbrowse add-track file.bed.gz --out /path/to/jbrowse2 --load copy
```

we will add that plaintext bed support to the cli soon though

**Reply by Opticalproperty** [reply]
Thank you for your response. I have successfully displayed my genomic information correctly, but there is still one question. When I embed jbrowse2 into my webpage, how can I default to opening a specific genetic locus that I want to display?

**Reply by cmdcolin** [reply, marked as answer]
See here https://jbrowse.org/jb2/docs/config_guides/default_session/ you can make a default session which will open the same thing every time. You can also use the URL param API to preparation link to particular places

**Comment by Opticalproperty** [comment]
Thanks for reply!
